ui-tests/src/test/java/org/openmrs/reference/page/ManageProviderSchedulesPage.java

```diff
@@ -15,6 +15,8 @@
 public class ManageProviderSchedulesPage extends Page {
 
     public static final By CURRENT_DAY = By.className("fc-state-highlight");
+    public static final By NEXT_WORK_DAY = By.xpath("/html/body/div/div[3]/div[2]/div[1]/div[2]/div/div/div/table/tbody/tr[1]/td[3]");
+    private static final By NEXT_MONTH = By.xpath("//*[@id=\"calendar\"]/table/tbody/tr/td[3]/span[4]/span");
```

Comment:
any reason why you want use the next month?

Reply:
I was trying to archive the next working day issue